### PR TITLE
fixed some technical terms in the readme

### PR DESCRIPTION
**Pull request by tropxy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ There are two recommended ways of running the project:
 
 ### Running with Docker
 Using Docker has the advantage of starting everything up automatically,
-including certificate generation, tests and linting, as well as creating
-both the SECC and EVCC.
+including certificate generation, tests and linting, as well as spawning
```
Comment (danielgordon-switch-ev) [inline README.md:20]: Either is fine in this context, I think -- @nikkigithub1 was there a specific reason to change "spawning" to "creating" in the first place?

File: README.md [hunk 2]
```diff
@@ -151,21 +151,22 @@ The SECC and EVCC have been tested together under:
 ## IPv6 WARNING
 
 For the system to work locally, the network interface needs to have
-an IPv6 link-local address assigned.
+an IPv6 local-link address assigned.
```
Comment (danielgordon-switch-ev) [inline README.md:154]: A quick Google search suggested that "link-local" is the correct one -- @tropxy are you sure about this?  According to Wikipedia, "In computer networking, a link-local address is a network address that is valid only for communications within the network segment or the broadcast domain that the host is connected to."

**Review by danielgordon-switch-ev — COMMENTED** [review]
I was wondering about a couple of these myself.

File: README.md (hunk 2 above)
Comment (tropxy) [reply README.md:154]: you are right, it is link-local

File: README.md [hunk 3]
```diff
@@ -192,21 +193,21 @@ The following table provides a few of the available variables:
 | MESSAGE_LOG_EXI   | `False`                       | Whether or not to log the EXI Bytestream messages (only works if log level is set to DEBUG)
 
 
-## Licence
+## License
```
Comment (danielgordon-switch-ev) [inline README.md:196]: I think this is fine -- American English is often the standard in a technical context.

**Review by danielgordon-switch-ev — APPROVED** [review]

**Pull request merged**